nautobot/extras/tests/test_filters.py

```diff
@@ -655,13 +655,13 @@ def test_grouping(self):
 
     def test_installed(self):
         params = {"installed": True}
-        # 31 local jobs and 3 plugin jobs
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 34)
+        # 33 local jobs and 4 plugin jobs
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 37)
 
     def test_enabled(self):
         params = {"enabled": False}
-        # 31 local jobs and 3 plugin jobs
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 34)
+        # 33 local jobs and 4 plugin jobs
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 37)
```

Comment:
Not blocking but it'd be nice to think of a way to change these tests so that they don't need to be updated every time we add a new test job.

Reply:
Can we just assert that a single matching job by name with installed set always returns a count? Same with enabled?

Reply:
Yeah, I think that'd be a good approach.